.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+ci:
+    autoupdate_schedule: 'monthly'
+    autofix_prs: false
+
+repos:
+-   repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black-jupyter
+-   repo: https://github.com/asottile/pyupgrade
+    rev: v3.3.1
+    hooks:
+    -   id: pyupgrade
+        args: [--py38-plus]
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0  # Use the ref you want to point at
+    hooks:
+    -   id: trailing-whitespace
```

Comment:
The CI fails due to the trailing whitespaces. This PR can be merged if this line is commented out, to be fixed in another PR.